pkg/autodiscovery/providers/etcd.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 )
 
+// Etcd represents the name of the config provider
+const Etcd = "etcd"
```

Comment:
We should probably capitalize this for consistency

Reply:
not sure for this one